package.json

```diff
@@ -41,11 +42,21 @@
     "jest": "^24.9.0",
     "jest-extended": "^0.11.5",
     "jest-transform-css": "^2.0.0",
+    "koa": "^2.11.0",
+    "koa-bodyparser": "^4.2.1",
+    "koa-ejs": "^4.3.0",
+    "koa-json": "^2.0.2",
+    "koa-jsonp": "^2.0.2",
+    "koa-logger": "^3.2.1",
+    "koa-override": "^3.0.0",
+    "koa-qs": "^2.0.0",
+    "koa-route": "^3.2.0",
+    "koa-send": "^5.0.0",
     "promise": "^8.1.0",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-test-renderer": "^16.12.0",
-    "recurly.js": "github:recurly/recurly-js",
+    "recurly.js": "github:recurly/recurly-js#export-test-server",
```

Comment:
Need to use this branch to expose the test server object so we can close it in jest.

Reply:
This will need to be master or a published version. Hopefully jest will close the server process on close, but if not then the server instance expose nees to be merged in rjs first.